google-cloud-bigquerystorage/src/main/java/com/google/cloud/bigquery/storage/v1/ProtoSchemaConverter.java

```diff
@@ -102,6 +102,12 @@ private static ProtoSchema convertInternal(
           enumTypes.add(enumFullName);
         }
       }
+      if (resultField.hasProto3Optional()) {
```

Comment:
nit: add some comment about why we need to clear the fields. Basically the binary string will be decoded as proto2 in the service side, and ProtoSchema needs to be proto2, so we need to clear proto3 only features. Proto2 and proto3 are binary compatible so it is OK.

Reply:
Done